Makefile

```diff
@@ -194,6 +194,10 @@ fmt:
 vet:
 	go vet ./...
 
+tidy: fmt vet
```

Comment:
wouldn't `lint` or `go-lint` be more descriptive ?

`tidy` reminds me of `go tidy` that fixes the dependencies, and not lints

Reply:
Fun fact: I started with `make lint` and then switched to `make tidy` :).

Oh well, be it `lint` then, fine with me.